src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/GaugeWave.cpp

```diff
@@ -0,0 +1,151 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/GaugeWave.hpp"
+
+#include <cmath>  // IWYU pragma: keep
+#include <cstdlib>
+#include <numeric>
+#include <ostream>
+#include <pup.h>
+#include <utility>
+
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/DataVector.hpp"        // IWYU pragma: keep
+#include "DataStructures/TempBuffer.hpp"
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Parallel/PupStlCpp11.hpp"
+#include "PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/StdArrayHelpers.hpp"
+#include "Utilities/StdHelpers.hpp"
+#include "Utilities/TMPL.hpp"
+
+// IWYU pragma: no_include <complex>
+
+// IWYU pragma: no_forward_declare Tensor
+
+/// \cond
```

Comment:
I don't think you need this \cond

Reply:
Surprisingly in many case we do, unfortunately. We think this is because of a bug in doxygen.

Reply:
I took the cond out, and I don't see a difference in the class reference page for GaugeWave. Does that mean in this case I really don't need it?